app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt

```diff
@@ -5,6 +5,8 @@ import android.content.Context
 import android.os.Handler
 import android.os.Looper
 import android.view.View
+import androidx.navigation.NavController
```

Comment:
Are all of these imports need? If not, please remove unused imports here & elsewhere in this PR.

Reply:
resolved